src/commands/HBonds.cpp

```diff
@@ -0,0 +1,206 @@
+// cfiles, an analysis frontend for the Chemfiles library
+// Copyright (C) 2015-2016 Guillaume Fraux
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+#include <docopt/docopt.h>
+#include <sstream>
+#include <fstream>
+
+#include "HBonds.hpp"
+#include "Errors.hpp"
+#include "utils.hpp"
+#include "geometry.hpp"
+
+using namespace chemfiles;
+
+static const double pi = 3.141592653589793238463;
+static const char OPTIONS[] =
+R"(Compute list of hydrogen bonds along a trajectory. A selection can be specified using the 
```

Comment:
"A selection can be specified .." => "Selections for the acceptor and donor atoms can be specified ..."